### PR TITLE
Feature price conversion decimal

### PR DESCRIPTION
**Pull request by witwitwut**

### CODE REVIEW

File: src/components/DataTable.vue [hunk 1]
```diff
@@ -33,16 +33,25 @@
...
+
     props: {
       bikes: Array
     },
 
     data() {
       return{
         search: '',
+
+        styling: 'text-xs-right',
```
Comment (klementj) [inline src/components/DataTable.vue]: Why the extra styling property?

File: src/components/DataTable.vue [hunk 2]
```diff
@@ -33,16 +33,25 @@
...
+
         header:[ 
           
           { text: 'Navn', align: 'left', value: 'title' },
```
Comment (klementj) [inline src/components/DataTable.vue]:
```suggestion
          { text: 'Name', align: 'left', value: 'title' },
```

File: src/components/DataTable.vue (hunk 1 above)
Comment (witwitwut) [reply src/components/DataTable.vue]: removed it, as it was just not that useful for reducing redundency

File: src/components/DataTable.vue (hunk 2 above)
Comment (witwitwut) [reply src/components/DataTable.vue]: I am not sure, but didn't we agree to have it all in danish? or are we going for a english version?

**Pull request merged**